tests/neo.UnitTests/UT_ProtocolSettings.cs

```diff
@@ -131,7 +32,7 @@ public void TestGetMemoryPoolMaxTransactions()
         [TestMethod]
         public void TestGetMillisecondsPerBlock()
         {
-            ProtocolSettings.Default.MillisecondsPerBlock.Should().Be(200);
+            ProtocolSettings.Default.MillisecondsPerBlock.Should().Be(15000);
```

Comment:
I think this will maybe affect the speed of dBFT tests.

Reply:
I think not, just checked that we removed this dependency some time ago and all tests are synchronous.